ios/RCTWeChat.m

```diff
@@ -238,13 +236,12 @@ - (void)shareToWeixinWithData:(NSDictionary *)aData
 
         } else if ([type isEqualToString:RCTWXShareTypeImageFile] || [type isEqualToString:RCTWXShareTypeImageResource]) {
             NSString * imageURL = aData[RCTWXShareImageUrl];
-            
             [self.bridge.imageLoader loadImageWithTag:imageURL callback:^(NSError *error, UIImage *image) {
                 if (image == nil){
                     callback(@[@"fail to load image resource"]);
                 } else {
                     WXImageObject *imageObject = [WXImageObject object];
-                    imageObject.imageData = RCTGetImageData([image CGImage], 1.0F);
+                    imageObject.imageData = UIImageJPEGRepresentation(image, 1.0F);
```

Comment:
This code may lead losing Alpha channel & transparency if the original image format is PNG.